docs/src/main/paradox/release-notes/releases-1.1.md

```diff
@@ -0,0 +1,55 @@
+
+### Bug Fixes
+
+* in TlsActor, flush data to user only after handshake has finished ([PR1128](https://github.com/apache/incubator-pekko/pull/1128))
+
+### Additional APIs
+
+* Add asInputStream to ByteString ([PR1085](https://github.com/apache/incubator-pekko/pull/1085))
+
+The Stream API has been updated to add some extra functions. 
+
+* add collectFirst stream operator ([PR984](https://github.com/apache/incubator-pekko/pull/984))
+* add collectWhile stream operator ([PR957](https://github.com/apache/incubator-pekko/pull/957))
+* add dimap stream operator ([PR942](https://github.com/apache/incubator-pekko/pull/942))
+* add flatten stream operator ([PR937](https://github.com/apache/incubator-pekko/pull/937))
+* add flattenMerge stream operator ([PR945](https://github.com/apache/incubator-pekko/pull/945))
+* add foldWhile stream operator ([PR1012](https://github.com/apache/incubator-pekko/pull/1012))
+* add mapAsyncPartitioned / mapAsyncPartitionedUnordered stream operators ([PR561](https://github.com/apache/incubator-pekko/pull/561))
+* add mapWithResource stream operator ([PR931](https://github.com/apache/incubator-pekko/pull/931), [PR1053](https://github.com/apache/incubator-pekko/pull/1053))
+* add onErrorComplete stream operator ([PR913](https://github.com/apache/incubator-pekko/pull/913))
+* add support for `for` comprehensions ([PR935](https://github.com/apache/incubator-pekko/pull/935))
+* add Sink.exists operator ([PR990](https://github.com/apache/incubator-pekko/pull/990))
+* add Sink.forall operator ([PR989](https://github.com/apache/incubator-pekko/pull/989))
+
+[Add More Details]
+
+### Removed
+
+* The pekko-protobuf jar is no longer published. The pekko-protobuf-v3 jar is still published ([PR489](https://github.com/apache/incubator-pekko/pull/489))
+
+### Other Changes
+
+* Scala 2 inline optimizer has been enabled
+* Classic Remoting was updated to use Netty 4 instead of Netty 3 ([PR643](https://github.com/apache/incubator-pekko/pull/643))
+* Support configuration for Jackson Recycler Pool ([PR1192](https://github.com/apache/incubator-pekko/pull/1192))
+* pekko-multi-node-testkit was changed to use Netty 4 instead of Netty 3 ([PR539](https://github.com/apache/incubator-pekko/pull/539))
+* add junit5 support to pekko-testkit-typed ([PR751](https://github.com/apache/incubator-pekko/pull/751))
+* Some performance changes in the Stream code ([PR48](https://github.com/apache/incubator-pekko/pull/48), [PR49](https://github.com/apache/incubator-pekko/pull/49), [PR278](https://github.com/apache/incubator-pekko/pull/278), [PR363](https://github.com/apache/incubator-pekko/pull/363), [PR408](https://github.com/apache/incubator-pekko/pull/408), [PR983](https://github.com/apache/incubator-pekko/pull/983), [PR1027](https://github.com/apache/incubator-pekko/pull/1027))
```

Comment:
https://github.com/apache/incubator-pekko/pull/252 is a behavior change, @mdedetrich , should be included into the release note too.

Reply:
adding now

Reply:
done